app/views/members/_form.html.erb

```diff
@@ -26,6 +26,10 @@
 <h3>Bio</h3>
   <%= form.text_area :bio %>
 
+<h3>Image</h3>
+<%= form.file_field :avatar%>
+
+<br></br>
```

Comment:
The closing tag here should be removed. Per [the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br):

> Must have a start tag, and must not have an end tag.

You might also consider a `<div>` here or a more semantic tag in place of the `br`.